docs/tutorial.md

```diff
@@ -51,10 +51,9 @@ that in a bit.)
 
 ## Creating our first change
 
-Now let's say we want to edit the `README.md` file in the repo (i.e. what you're
```

Comment:
Hmm, not sure why this is showing up here? This change was already merged...

Reply:
And... it's gone. nvm 😅

Reply:
The commit is actually still there, but I assume it will be dropped when I merge this PR because it matches what's already at head.

Reply:
Oh, you accidentally put the changelog update in a commit called "[Remove out-of-date parenthetical in tutorial.md](https://github.com/martinvonz/jj/commit/ad79d89e4baa534879cffe43c6d9a0251303c938)" :P Oh well, not the end of the world

Reply:
Hmm, whoops... Still getting used to working with `jj`, and I thought I squashed it into the previous commit. :/

Reply:
Ah, that would explain it. I suspect you did `jj squash -r @-`, thinking that would squash *into* that commit, but it actually squash *from* that commit and into its parents. What you wanted instead was `jj squash -r @` (to squash the working copy into its parent), which is the default, so just `jj squash` is enough (or `jj amend`, an alias, if you prefer). By the way, if you actually had wanted to move the change into the grandparent, that would be `jj move --to @--`. `jj move` is a really useful command that I've been using more and more recently.